src/BgZ-3-0/Cert/_reference/resources/medmij-bgz-encounter-ts-02.xml

```diff
@@ -0,0 +1,46 @@
+<?xml-model href="http://hl7.org/fhir/STU3/encounter.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><Encounter xmlns="http://hl7.org/fhir" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hl7.org/fhir http://hl7.org/fhir/STU3/encounter.xsd">
+   <id value="medmij-bgz-encounter-ts-02"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-Encounter"/>
+   </meta>
+   <identifier>
+      <system value="urn:oid:2.16.840.1.113883.2.4.3.11.999.7.6"/>
+      <value value="2f8758ec-e3e4-11eb-1081-020000000000"/>
+   </identifier>
+   <status value="finished"/>
+   <class>
+      <system value="http://hl7.org/fhir/v3/ActCode"/>
+      <code value="IMP"/>
+      <display value="inpatient encounter"/>
+   </class>
+   <subject>
+      <reference value="Patient/medmij-bgz-patient-ts-01"/>
+      <display value="Johan XXX_Helleman"/>
+   </subject>
+   <participant>
+      <individual>
+         <extension url="http://nictiz.nl/fhir/StructureDefinition/practitionerrole-reference">
+            <valueReference>
+               <reference value="PractitionerRole/medmij-bgz-practitionerrole-ts-03"/>
+               <display value="Orthopedie"/>
+            </valueReference>
+         </extension>
+         <reference value="Practitioner/medmij-bgz-practitioner-ts-01"/>
+         <display value="G.Z.M. de Wit"/>
+      </individual>
+   </participant>
+   <period>
+      <start value="2012-08-16"/>
+      <end value="2012-08-19"/>
+   </period>
+   <diagnosis>
+      <condition>
+         <reference value="Procedure/medmij-bgz-procedure-ts-02"/>
+         <display value="Surgical reduction of fracture (procedure)"/>
```

Comment:
Kunnen we deze display-waarde ook meteen in het Nederlands neerzetten (net als in Procedure-resource zelf)?